src/scores/continuous.py

```diff
@@ -6,31 +6,35 @@
 
 
 def mse(fcst, obs, reduce_dims=None, preserve_dims=None, weights=None):
-    """
+    """Calculates the mean squared error from forecast and observed data.
 
-    Returns:
-      - By default an xarray containing a single floating point number representing the mean absolute
-        error for the supplied data. All dimensions will be reduced.
-      - Otherwise: Returns an xarray representing the mean squared error, reduced along
-      the relevant dimensions and weighted appropriately.
+    Dimensional reduction is not supported for pandas and the user should
+    convert their data to xarray to formulate the call to the metric. At
+    most one of reduce_dims and preserve_dims may be specified.
+    Specifying both will result in an exception.
 
     Args:
-      - fcst: Forecast or predicted variables in xarray or pandas
-      - obs: Observed variables in xarray or pandas
-      - reduce_dims: Optionally specify which dimensions to reduce when calculating MSE.
-                     All other dimensions will be preserved.
-      - preserve_dims: Optionally specify which dimensions to preserve when calculating MSE. All other
-                       dimensions will be reduced. As a special case, 'all' will allow all dimensions to
-                       be preserved. In this case, the result will be in the same shape/dimensionality as
-                       the forecast, and the errors will be the squared error at each point (i.e. single-value
-                       comparison against observed), and the forecast and observed dimensions must match
-                       precisely.
-      - weights: Not yet implemented. Allow weighted averaging (e.g. by area, by latitude, by population, custom)
-
-    Notes:
-      - Dimensional reduction is not supported for pandas and the user should convert their data to xarray
-        to formulate the call to the metric.
-      - At most one of reduce_dims and preserve_dims may be specified. Specifying both will result in an exception.
+        fcst: Forecast or predicted variables in xarray or pandas
+        obs: Observed variables in xarray or pandas
```

Comment:
Union[xr.Dataset, xr.DataArray, pd.Dataframe, pd.Series]

Reply:
fixed in 72ea4e1